coresdk/src/coresdk/logging.h

```diff
@@ -0,0 +1,51 @@
+#ifndef logging_h
+#define logging_h
+
+#include "terminal.h"
+#include <chrono>
+#include <ctime>
+#include <fstream>
+
+namespace splashkit_lib
+{
+
+  /**
+   *  
+   * @constant INFO     Output information to the message log
+   * @constant DEBUG    Output a debug message to the message log
+   * @constant WARNING  Output a warning message to the message log
+   * @constant ERROR    Output an error message to the message log
+   */
+  enum log_level {
```

Comment:
Order these in ascending order - so most detailed log level first.

Reply:
I would generally have DEBUG as more detail than INFO

Reply:
When you reorder this you also need to reorder the comments. From memory the translator can get this wrong if the orders are different.